wile.py

```diff
@@ -43,7 +44,12 @@ def wile(obj, directory_url, staging, account_key_path, new_account_key_size, ve
 
     if staging:
         directory_url = 'https://acme-staging.api.letsencrypt.org/directory'
-    account_key = get_or_gen_key(account_key_path, new_account_key_size)
+
+    try:
+        account_key = get_or_gen_key(account_key_path, new_account_key_size)
+    except ValueError as e:
+        logger.error('regarding %s: %s' % (account_key_path, str(e)))
```

Comment:
I copied the way that `logger.*(...)` is used in the current code. I don't find a single use with comma the way you ask for so I'm a bit puzzled by that request, but sure I can fix that.

I'm happy to put in any message you like, I felt this one would work. Please propose a better message.